packages/eds-core-react/src/components/Typography/Typography.stories.tsx

```diff
@@ -223,57 +83,62 @@ export const colors = (): JSX.Element => (
     <Typography color="disabled">Disabled</Typography>
     <Typography color="currentColor">CSS currentColor</Typography>
     <Typography color="pink">CSS pink</Typography>
-  </Grid>
+  </>
 )
 
-export const custom = (): JSX.Element => (
-  <Grid>
-    <Typography group="navigation" variant="label">
-      Navigation / Label
-    </Typography>
-    <Typography group="navigation" variant="menu_title">
-      Navigation / Menu / Title
-    </Typography>
-    <Typography group="table" variant="cell_text">
-      Table / Cell / Text
-    </Typography>
-    <Typography
-      group="table"
-      variant="cell_text"
-      token={{ fontWeight: 'medium' }}
-    >
-      Table / Cell / Text / Medium
-    </Typography>
-    <Typography group="table" variant="cell_text" bold>
-      Table / Cell / Text / Bold
-    </Typography>
-    <Typography group="table" variant="cell_text" link href="#">
-      Table / Cell / Text / Bold / Link
-    </Typography>
-
-    <Typography variant="h3" token={{ fontWeight: 700 }}>
-      Heading 3 Bold
-    </Typography>
-    <Typography variant="ingress" token={{ fontWeight: 'bold' }}>
-      Ingress Bold
-    </Typography>
-    <Typography
-      token={{
-        color: 'purple',
-        fontFamily: 'Arial',
-        fontSize: '1.875rem',
-        fontWeight: 300,
-        lineHeight: '1.714em',
-        textTransform: 'uppercase',
-      }}
-    >
-      Custom token
-    </Typography>
-  </Grid>
+export const Custom: Story<TypographyProps> = () => (
```

Comment:
Maybe change this story to be about the token property? Its not really custom as its just examples of different combinations using props.

Reply:
done